scripts/backup

```diff
@@ -1,30 +1,86 @@
 #!/bin/bash
-set -eu
 
-app=$YNH_APP_INSTANCE_NAME
+#=================================================
+# GENERIC START
+#=================================================
+# IMPORT GENERIC HELPERS
+#=================================================
 
-# Source YunoHost helpers
+source ../settings/scripts/_common.sh
 source /usr/share/yunohost/helpers
 
+#=================================================
+# MANAGE SCRIPT FAILURE
+#=================================================
+
+ynh_clean_setup () {
+	### Remove this function if there's nothing to clean before calling the remove script.
+	true
+}
+# Exit if an error occurs during the execution of the script
+ynh_abort_if_errors
+
+#=================================================
+# LOAD SETTINGS
+#=================================================
+
+app=$YNH_APP_INSTANCE_NAME
+
 final_path=$(ynh_app_setting_get $app final_path)
-statedir=$(ynh_app_setting_get $app statedir)
-final_logpath=$(ynh_app_setting_get $app final_logpath)
-final_nginxconf=$(ynh_app_setting_get $app final_nginxconf)
-final_phpconf=$(ynh_app_setting_get $app final_phpconf)
+domain=$(ynh_app_setting_get $app domain)
+#db_name=$(ynh_app_setting_get $app db_name)
+config_path=$(ynh_app_setting_get $app config_path)
+state_path=$(ynh_app_setting_get $app state_path)
```

Comment:
statedir or state_path ?

Reply:
values are:
```
final_path=/usr/share/z-push
config_path=/etc/z-push
state_path=/var/lib/z-push
log_path=/var/log/z-push
```

is it better to name variable state_dir or state path ?

Reply:
It's better to keep the same name than before, to not have to do a migration.